app/behaviour.py

```
@@ -98,7 +100,15 @@ def __test_strategies(self, market_data):
                                     )
 
                         else:
-                            self.logger.warn("No such behaviour: %s, skipping.", behaviour)
+                            self.logger.warn("No such behaviour %s, skipping.", behaviour)
+                except ValueError as e:
```

I think it would be best to just let the ValueError be thrown here, as it provides useful information in itself